managed/src/main/java/com/yugabyte/yw/controllers/CertificateController.java

```diff
@@ -128,13 +129,34 @@ public Result getRootCert(UUID customerUUID, UUID rootCA) {
     }
   }
 
-  public Result list(UUID customerUUID) {
-    List<CertificateInfo> certs = CertificateInfo.getAll(customerUUID);
-    if (certs == null) {
-      return ApiResponse.error(BAD_REQUEST, "Invalid Customer UUID: " + customerUUID);
-    }
-    return ApiResponse.success(certs);
-  }
+	public Result list(UUID customerUUID) {
+		List<CertificateInfo> certs = CertificateInfo.getAll(customerUUID);
+		HashMap<String, Boolean> removable = new HashMap<String, Boolean>();
+		List<Universe> universes = Universe.getAll();
+		for (Universe universe : universes) {
+			try {
+				Universe universe_obj = Universe.get(universe.universeUUID);
+				UUID cert = universe_obj.getUniverseDetails().rootCA;
+				removable.put(cert.toString(), true);
+			} catch (NullPointerException a) {
+				continue;
+			}
+		}
+		JsonNode certificates = Json.toJson(certs);
+		for (JsonNode cert : certificates) {
+			JsonNode cert_uuid = cert.get("uuid");
+			Boolean value = removable.get(cert_uuid.asText());
+			if (value == null) {
+				((ObjectNode) cert).put("removable", true);
+			} else {
+				((ObjectNode) cert).put("removable", false);
```

Comment:
I'd suggest renaming this prop as it's unclear why a cert could be marked as removable
smth like `inUse` will be self-explanatory